crates/ef-testing/src/models/case.rs

```diff
@@ -136,14 +137,9 @@ impl BlockchainTestCase {
         )?;
 
         let client = env.client();
-        let hash = client.send_transaction(tx_encoded).await?;
-
-        // we make sure that the transaction has a receipt and fail fast if it doesn't
-        let starknet_provider = env.client().starknet_provider();
-        let transaction_hash: FieldElement = FieldElement::from_bytes_be(&hash)?;
-        starknet_provider
-            .get_transaction_receipt::<FieldElement>(transaction_hash)
-            .await?;
+        // Send the transaction without checking for errors, accounting
+        // for the fact that some transactions might fail.
+        let _ = client.send_transaction(tx_encoded).await;
```

Comment:
```suggestion
        client.send_transaction(tx_encoded).await;
```

Reply:
btw why 
```        // Send the transaction without checking for errors, accounting
```

Reply:
The linter was telling me to add the `let _`, because send_transaction sends back an error which I don't want to handle. This error comes from the fact that the transaction waiter we have in the RPC will send an error if the transaction he is awaiting returns a rejected transaction.